src/Services/cdl/CdlHeader.js

```
@@ -12,75 +12,97 @@ class CdlHeader {
+
+            // Get logtype info for current logtype
+            const ltInfo = this.header.ltMap[logtype];
+            if (!ltInfo?.funcid === undefined || !Array.isArray(ltInfo.vars)) {
+                throw new Error(`Invalid ltInfo structure for logtype: ${logtype}.`);
+            }
+
+            // Add to logtype map
+            this.logTypeMap[logtype] = new LtInfo(ltInfo, ltInfo.funcid, fileName);
+
+            // Add to function map
+            !(ltInfo.funcid in this.functionMap) && (this.functionMap[ltInfo.funcid] = []);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)